bindata/etcd/restore-pod.yaml

```diff
@@ -24,6 +24,27 @@ spec:
         export ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME}
         export ETCD_INITIAL_CLUSTER="${ETCD_NAME}=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380"
         env | grep ETCD | grep -v NODE
+        export ETCD_NODE_PEER_URL=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380
+
+        # checking if data directory is empty, if not etcdctl restore will fail
+        if [ ! "$(ls -A /var/lib/etcd)" ]; then
+          echo "please delete the data directory before restoring"
+          exit 0
+        fi
+
+        # checking if we have backup file to be restored
+        if [ ! -f var/lib/etcd-backup/snapshot.db ]; then
+            echo "please copy the backup file to /var/lib/etcd-backup/snapshot.db before restoring"
+            exit 0
+        fi
+
+        echo "restoring to a single node cluster"
+        ETCDCTL_API=3 etcdctl snapshot restore /var/lib/etcd-backup/snapshot.db \
```

Comment:
I am not sure if we would have access to etcdctl in PATH, probably will have to modify with the path to binary for preciseness